hi-ml-cpath/testhisto/testhisto/utils/test_viz_utils.py

```diff
@@ -185,8 +184,8 @@ def test_plot_attention_tiles_below_min_rows(slide_node: SlideNode, caplog: LogC
 def test_plot_slide(test_output_dirs: OutputFolderForTests, scale: int) -> None:
     set_random_seed(0)
     slide_image = np.random.rand(3, 1000, 2000)
-    slide_node = SlideNode(slide_id="slide_0", prob_score=0.5, true_label=1, pred_label=1)
-    fig = plot_slide(case="TP", slide_node=slide_node, slide_image=slide_image, scale=scale)
+    slide_node = SlideNode(slide_id="slide_0", gt_prob_score=0.04, pred_prob_score=0.96, true_label=1, pred_label=0)
```

Comment:
for clarity - I think it's enough to parametrize also this input to have one FN and one TP at least. In reality the only assertion for which I think is worth to parametrize is `assert_binary_files_match`

Reply:
Discussed offline. We added few more cases for TP TN FN FP